dev-util/flawfinder/flawfinder-2.0.7.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_6 )
```

Comment:
Please consider adding python 3_5 and maybe even 3_7.

Reply:
```diff
-PYTHON_COMPAT=( python2_7 python3_6 )
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```